backend/app.js

```diff
@@ -24,7 +24,11 @@ app.use(urlencodedParser)
 app.use(jsonParser)
 
 let bot 
-botFactory.makeBot(process.env.FOOSBOT_TOKEN, process.env.FOOS_CHANNEL_NAME)
+botFactory.makeBot(
```

Comment:
prefix suffix assignment is not a part of the bot's respobnsibility. bot should only be able to post messages to a channel. it's the responsibility of the match reporter (different file). please don't put it in the bot